diff --git a/dist/tua-bsl.mjs b/dist/tua-bsl.mjs
index ab83570cdd70766c7b72f0e5293a532f867458bc..c8b9c868855e35088d183e6fefe9e6686d925b4d 100644
--- a/dist/tua-bsl.mjs
+++ b/dist/tua-bsl.mjs
@@ -178,13 +178,13 @@ function setOverflowForPc(options) {
   var $html = document.documentElement;
   var htmlStyle = Object.assign({}, $html.style);
   var withPaddingRight = (_a = void 0 ) !== null && _a !== void 0 ? _a : true;
-  $html.style.overflow = 'hidden';
-  $html.style.boxSizing = 'border-box';
   if (withPaddingRight) {
     var scrollBarWidth = window.innerWidth - $html.clientWidth;
     var previousPaddingRight = parseInt(window.getComputedStyle($html).paddingRight, 10);
     $html.style.paddingRight = "".concat(scrollBarWidth + previousPaddingRight, "px");
   }
+  $html.style.overflow = 'hidden';
+  $html.style.boxSizing = 'border-box';
   return function () {
     ['overflow', 'boxSizing', 'paddingRight'].forEach(function (x) {
       $html.style[x] = htmlStyle[x] || '';
